torchrl/modules/models/models.py

```diff
@@ -271,18 +292,20 @@ class ConvNet(nn.Sequential):
             depth, defined by the num_cells or depth arguments.
         strides (int or Sequence[int]): Stride(s) of the conv network. If iterable, the length must match the
             depth, defined by the num_cells or depth arguments.
-        activation_class (Type): activation class to be used.
+        activation_class (Type[nn.Module]): activation class to be used.
             default: nn.Tanh
         activation_kwargs (dict, optional): kwargs to be used with the activation class;
         norm_class (Type, optional): normalization class, if any;
         norm_kwargs (dict, optional): kwargs to be used with the normalization layers;
         bias_last_layer (bool): if True, the last Linear layer will have a bias parameter.
             default: True;
-        aggregator_class (Type): aggregator to use at the end of the chain.
+        aggregator_class (Type[nn.Module]): aggregator to use at the end of the chain.
             default:  SquashDims;
         aggregator_kwargs (dict, optional): kwargs for the aggregator_class;
         squeeze_output (bool): whether the output should be squeezed of its singleton dimensions.
             default: True.
+        device (DEVICE_TYPING): device to create the module on.
```

Comment:
we should not set `self.device` in the class as this will be inconsistent when calling `module.to(device)`